lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +265,17 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        errors = []
+        command = "systemctl restart #{service}"
```

Comment:
this will break the case when user enters
```
$ vagrant service-manager restart k8s
```

`k8s` is not a systemd service name. If one wants to restart k8s, there are bunch of services needed to be configured.

Check steps to configure kuberentes https://github.com/projectatomic/adb-utils/blob/master/utils/sccli.sh#L6

Also, we should provide the option for user to either say `k8s` or `kubernetes` and we understand it.